lib/gutentag/tag_names.rb

```diff
@@ -1,21 +0,0 @@
-class Gutentag::TagNames
-  def self.call(*names)
-    new(names).call
-  end
-
-  def initialize(names)
-    @names = names.flatten
-  end
-
-  def call
-    name_values.collect { |name| Gutentag::TagName.call name }
```

Comment:
ah ... I think I get what you had in mind ... make TagNames be 
```
tags.map { |t| Gutentag.normaliser.call(t.is_a?(Gutentag::Tag) ? t.name : t) }
```
... but that pattern does not make sense to me ... just a random method ... no need to wrap it into a class ...
if we followed that pattern then every method should be a class ... and that would be madness ...